webapp_provider/nginx.conf

```diff
@@ -0,0 +1,39 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+
+    include    /etc/nginx/mime.types;
+    default_type application/octet-stream;
+    access_log   /var/log/nginx/access.log;
+    sendfile on;
+
```

Comment:
maybe we need to add default timeouts and upload body size. otherwise users cannot upload large files right?